.github/workflows/test.yml

```diff
@@ -454,35 +454,11 @@ jobs:
         working-directory: crates/turbopack-ecmascript-runtime/js
         run: pnpm run check
 
-  rust_prepare:
-    name: Check rust crates
-    runs-on: ubuntu-latest-16-core-oss
-    needs: determine_jobs
-    if: needs.determine_jobs.outputs.rust == 'true'
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-
-      - name: Setup Go
-        uses: ./.github/actions/setup-go
-        with:
-          github-token: "${{ secrets.GITHUB_TOKEN }}"
-
-      - name: Setup Rust
-        uses: ./.github/actions/setup-rust
-        with:
-          save-cache: true
-
-      - name: Run cargo check
-        uses: actions-rs/cargo@v1
-        with:
-          command: check
-          args: --workspace --all-targets --locked --features rustls-tls
-
-  rust_lint:
-    needs: [determine_jobs, rust_prepare]
+  # Very quick checks that we want to run first before all the long-running checks.
+  rust_pre_check:
```

Comment:
merging these now means there's an extra 5 minute delay while it's checking licenses :/

Reply:
And that format errors now block tests from running